src/bin/lib/config.ts

```diff
@@ -1,5 +1,6 @@
 import * as path from 'path'
 import * as os from 'os'
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
`uuid` only exists as a transitive dependency, please make it a direct dependency.